File: dna/android/ExoPlayer-pre2.10-Java/app/src/main/java/io/streamroot/dna/samples/exoplayer/SRApplication.java
package io.streamroot.dna.samples.exoplayer;

import androidx.multidex.MultiDexApplication;

import io.streamroot.dna.core.DnaClient;

public final class SRApplication extends MultiDexApplication {
    @Override
    public void onCreate() {
        super.onCreate();
        DnaClient.initializeApp(this);
    }
}

File: dna/android/ExoPlayer/app/src/main/java/io/streamroot/dna/samples/exoplayer/SRApplication.kt
package io.streamroot.dna.samples.exoplayer

import androidx.multidex.MultiDexApplication
import io.streamroot.dna.core.DnaClient

class SRApplication: MultiDexApplication() {
    override fun onCreate() {
        super.onCreate()
        DnaClient.initializeApp(this)
    }
}